lighthouse-core/audits/seo/hreflang.js

```diff
@@ -57,16 +41,17 @@ function isFullyQualified(href) {
 
 /**
  * @param {string} hreflang
+ * @param {(lang: string) => boolean} isValidLang From axe.
  * @return {boolean}
  */
-function isExpectedLanguageCode(hreflang) {
+function isExpectedLanguageCode(hreflang, isValidLang) {
```

Comment:
Any reason to make this a callback? Could import where needed:
```suggestion
async function isExpectedLanguageCode(hreflang) {
  // TODO(esmodules): use static import when file is esm.
  const isValidLang = (await import('../../../third-party/axe/valid-langs.js')).default;
  
  // ...
```

Reply:
I was trying to limit the async from going where it made less sense.